### DISCUSSION TITLE
Adding Packs into Fleet using Fleetctl

### DISCUSSION THREAD

**Discussion by ow111** [Security-Onion-Solutions/securityonion#4064] [Unsupported Versions] [answered]
Hi,
On my manager node in a distributed deployment I am trying to add some extra packs. (20+)
I cloned a repo with the .conf files and used fleetctl to convert these to the .yaml format - that worked ok.
Now wanting to get these .yaml files into fleet by using `fleetctl apply -f`

I have setup fleetctl with the correct web addr  - https://my-mgr-node:8080 and rootca -` /etc/pki/fleet.crt`
When i try to login with user credentials or credentials from "salt-call pillar.get secrets" I get the error : The Fleet instance is not set up yet. 
Am i doing something wrong here

**Comment by ow111** [comment]
Is this the correct way to add extra packs into Fleet?

**Comment by defensivedepth** [comment, marked as answer]
@ow111 

I added a section to the docs describing how to connect via fleetctl.

https://docs.securityonion.net/en/2.3/fleet.html#fleetctl

**Comment by Uzzi** [comment]
Hi can yo help about:
`sudo docker exec -it so-fleet fleetctl package --type=msi --fleet-url=https://<securityoninionIP>:8080 --enroll-secret=8xxxxxxxxxxxxxx
Generating your osquery installer...
Error: package root files: heat failed: exec: "docker": executable file not found in $PATH`

**Reply by defensivedepth** [reply]
@Uzzi What are you attempting to do?

Also - please start a new Discussion.
